scripts/orphaned.py

```
@@ -33,42 +72,122 @@ def check_files_on_disk(client, torrents: List, exclude_file_patterns: List[str]
+                if entry_resolved in torrent_files:
                     continue  # Skip files that are tracked by torrents
+
                 orphaned_files.append(str(entry))
 
+    logging.info(f"Scanned {files_checked} files, excluded {files_excluded_by_pattern} by pattern, "
+                 f"excluded {files_excluded_by_dir} by directory")
+
     return orphaned_files
 
-def delete_orphaned_files(orphaned_files: List[str], dry_run: bool, client):
+def delete_orphaned_files(orphaned_files: List[str], dry_run: bool, client, torrents: List = None):
```

Ruff (0.14.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)